docs/cases/api/cases-api/cases-api-add-comment.asciidoc

```diff
@@ -22,13 +21,25 @@ A JSON object with a `comment` field:
 |==============================================
 |Name |Type |Description |Required
 
-|`type` |String |The comment type, which can be `user` or `alert`. |Yes
-|`comment` |String |The case's new comment. |Yes, only when type is `user`.
-|`alertId` |String |The alert id. |Yes, only when type is `alert`.
-|`index` |String |The alert index. |Yes, only when type is `alert`.
+|`alertId` |String |The alert identifier. |Yes, only required when the  `type` is `alert`.
```

Comment:
```suggestion
|`alertId` |String |The alert identifier. |Yes, only when the  `type` is `alert`.
```

Reply:
In this case, "required" is redundant so I can remove it across the table.